.gitignore

```diff
@@ -331,3 +331,7 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+# Remove generated files
```

Comment:
>generated [](start = 9, length = 9)

If these files are generated, why we have them checked in? The client script lib should come from npm dependency at build time.

Reply:
Removed the changes.